.travis.yml

```diff
@@ -1,26 +1,42 @@
 language: generic
+dist: trusty
```

Comment:
I guess it doesn't matter. In any case, in Travis, if you don't set it otherwise, it will use Precise, instead of Trusty (or a newer version).